### ISSUE TITLE
Add option to change mouse sensibility

### ISSUE THREAD

**Issue by mattboll** [fyne-io/fynedesk#206]
I'm using a touchpad right now and the mouse speed is slow, I need to move my finger several times through the touchpad to go from a border to another

**Comment by andydotxyz** [comment]
Thanks. I guess you mean sensitivity though? :)

**Comment by mattboll** [comment]
yes, excuse my french ^^ Actually, I guess it's a combo of sensitivity and acceleration

**Comment by andydotxyz** [comment]
Cool. I think you're right it is a bit of both
